src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -288,8 +287,16 @@ public synchronized static AccordService startup(NodeId tcmId)
 
         AccordService as = new AccordService(AccordTopology.tcmIdToAccord(tcmId));
         unsafeInstance = as;
-        as.startup();
-        replayJournal(as);
+        as.node.unsafeSetReplaying(true);
+        try
+        {
+            as.startup();
```

Comment:
I don't think this should matter, but maybe it's worth to move `startup` out from here? I think the issue was only with log replay.